xbmc/windows/GUIWindowHome.cpp

```diff
@@ -74,11 +77,10 @@ void CGUIWindowHome::Announce(ANNOUNCEMENT::AnnouncementFlag flag, const char *s
   if (data.isMember("transaction") && data["transaction"].asBoolean())
     return;
 
-  if (strcmp(message, "OnScanStarted") == 0 ||
-      strcmp(message, "OnCleanStarted") == 0)
+  if (message == "OnScanStarted" || message == "OnCleanStarted")
     return;
 
-  bool onUpdate = strcmp(message, "OnUpdate") == 0;
+  bool onUpdate = message == "OnUpdate";
```

Comment:
Please use brackets like (message == "OnUpdate"), it improves readability. And personally I would mark onUpdate const.

Reply:
We have those hardcoded strings all over and I don't want to "pollute" this PR with changing another 100 of string literal replaces - I'll probably do that in a separate PR.
I prefer to not have the brackets because I think we use that style on a lot of other places as well.